### PR TITLE
Add ovidiutirla to sig-foundations

### PR DESCRIPTION
**Pull request by ovidiutirla**
Hey! 
In the past few months I've contributed to the sig-foundations area, and I would like to keep contributing and take the responsibility of a reviewer. 

Some of the contribution in this area:
https://github.com/cilium/hive/pulls?q=is%3Apr+is%3Aclosed+author%3Aovidiutirla
https://github.com/cilium/cilium/pulls?q=is%3Apr+author%3Aovidiutirla+
including some PR reviewers 
CFPs:
https://github.com/cilium/cilium/issues/33502
https://github.com/cilium/cilium/issues/34101
https://github.com/cilium/cilium/issues/34614
https://github.com/cilium/cilium/issues/34823
https://github.com/cilium/cilium/issues/34541

### CODE REVIEW

**Comment by ovidiutirla** [comment]
Thanks, @cilium/sig-foundations 😄

File: ladder/teams/sig-foundations.yaml [hunk 1]
```diff
@@ -5,3 +5,4 @@ members:
 - joamaki
 - pippolo84
 - tommyp1ckles
+- ovidiutirla
```
Comment (bimmlerd) [inline ladder/teams/sig-foundations.yaml:8]: nit: could be a coincidence, but I think the list is alphabetic

**Review by bimmlerd — APPROVED** [review]
I'm all for it. Feel free to always ping others if code touches an area your unfamiliar with (if those still exist 😄 )

**Review by tommyp1ckles — APPROVED** [review]
Based on what I've seen reviewing @ovidiutirla 's recent contributions, I give this a thumbs up.

**Pull request merged**